lib/i18n/js/private/config_store.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "singleton"
+
+module I18n
+  module JS
+    # @api private
+    module Private
+      # Caching implementation for I18n::JS.config
+      #
+      # @api private
+      class ConfigStore
+        include Singleton
+
+        def fetch
+          return @config if @config
+
+          yield.tap do |obj|
+            raise AugumentError, "unexpected falsy object from block" unless obj
```

Comment:
Hello. Is there an `AugumentError` exception?

Reply:
There is:
https://ruby-doc.org/core-2.7.0/ArgumentError.html

Reply:
I'm aware but there is a typo here. :)

Reply:
Arrr
Let me fix

Reply:
Thanks.